docs/tutorials/generic_field_type/3_create_form_for_point2d.md

```diff
@@ -0,0 +1,118 @@
+# Step 3 - Create form for editing Field Type
+
+!!! tip
+
+    You can find all files used and modified in this step on [GitHub]().
+
+## Create a form
+
+To edit your new Field Type, create a `Point2DType.php` form in the `src/Form/Type` directory.
+Next, add a `Point2DType` class that extends the `AbstractType` and implements the `buildForm()` method.
+This method adds fields for `x` and `y` coordinates.
+It also enables you to skip implementation of Data Transformer.
+
+```php
+<?php
+declare(strict_types=1);
+
+namespace App\Form\Type;
+
+use App\FieldType\Point2D\Value;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class Point2DType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add('x', NumberType::class);
+        $builder->add('y', NumberType::class);
+    }
+}
+```
+
+## Add a Form Mapper Interface
+
+The FormMapper adds the Field definitions into Symfony forms using the `add()` method. 
+You will add it to be able to edit the fields later on.
+For more information about the FormMappers, see [Field Type form and template](../../api/field_type_form_and_template.md).
+
+First, add a `FieldValueFormMapperInterface` interface (`EzSystems\RepositoryForms\FieldType\FieldValueFormMapperInterface`) to Field Type definition in the `src/FieldType/Point2D/Type.php`.
+
+Next, add a `mapFieldValueForm()` with the following arguments (highlighted lines):
+
+- Name of the property the Field value will map to: `value`
+- Type of the Field: `Point2DType::class`
+- Custom options: `required` and `label`
```

Comment:
Sorry, but I don't understand this part. You mean `FieldValueFormMapperInterface ::mapFieldValueForm` or `FormInterface::add` ?